includes/wp-admin/class-settings-fields.php

```diff
@@ -466,4 +482,27 @@ public static function render_authorized_fetch_field() {
 		</p>
 		<?php
 	}
+
+	/**
+	 * Render relays field.
+	 */
+	public static function render_relays_field() {
+		$value = get_option( 'activitypub_relays', array() );
+		?>
+		<p>
+			<?php echo wp_kses( __( 'A <strong>Fediverse-Relay</strong> distributes content across instances, expanding reach, engagement, and discoverability, especially for smaller instances.', 'activitypub' ), 'default' ); ?>
+		</p>
+		<textarea
+			id="activitypub_relays"
+			name="activitypub_relays"
+			class="large-text"
+			cols="50"
+			rows="5"
+		><?php echo esc_textarea( implode( PHP_EOL, $value ) ); ?></textarea>
+		<p class="description">
+			<?php echo wp_kses( __( 'Enter the <strong>Inbox-URLs</strong> (e.g. <code>https://relay.example.com/inbox</code>) of the relays you want to use, one per line.', 'activitypub' ), array( 'strong' => array() ) ); ?>
```

Comment:
The allowlist will need `code`, too, unless you want to add it through a placeholder